scripts/beachball/tagVNext.ts

```diff
@@ -0,0 +1,70 @@
+import { execSync } from 'child_process';
+import * as semver from 'semver';
+import * as tmp from 'tmp';
+import * as fs from 'fs';
+import { AllPackageInfo, getAllPackageInfo, isConvergedPackage } from '../monorepo/index';
+
+function tagPackages() {
+  const packagesToTag = getPackagesToTag();
+  const { path: npmrcPath, cleanup: cleanupTmpFiles } = createTmpNpmrc();
+
+  packagesToTag.forEach(pkg => {
+    tagPackage(pkg.name, pkg.version, npmrcPath);
+  });
+
+  cleanupTmpFiles();
+}
+
+function tagPackage(name: string, version: string, npmrcPath: string) {
+  const prerelease = semver.parse(version).prerelease;
```

Comment:
Mainly curious, what’s the case where you can have more than one prerelease in a version?

Reply:
`alpha.1` is actually parsed as `[alpha, 1]`